pkg/workload/tpcc/tpcc.go

```diff
@@ -620,13 +621,22 @@ func (w *tpcc) Ops(
 	w.reg = reg
 	w.usePostgres = parsedURL.Port() == "5432"
 
+	method, err := workload.StringToMethod(w.connFlags.Method)
+	if err != nil {
+		return workload.QueryLoad{}, err
+	}
+
+	// We only Prepare statements if Prepare is explicitly specified.
+	noPrepare := method != workload.Prepare
```

Comment:
i think the condition should be `method == workload.Simple` (based on my previous comment)